app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/singledisposals/summary.scala.html

```diff
@@ -100,16 +109,26 @@
             "propertyType"
         )
     }
-  @cyaRow(
-    messages(s"disposalDate$userKey.title"),
-    Html(TimeUtils.govDisplayFormat(answers.disposalDate.value)),
-    Some(cyaChange(messages("disposalDate.cyaChange"), routes.SingleDisposalsTriageController.whenWasDisposalDate().url)),
-      "disposalDate"
-  )
-  @cyaRow(
-    messages(s"completionDate$userKey.title"),
-    Html(TimeUtils.govDisplayFormat(answers.completionDate.value)),
-    Some(cyaChange(messages("completionDate.cyaChange"), routes.SingleDisposalsTriageController.whenWasCompletionDate().url)),
-      "completionDate"
-  )
+
+    @if(isSingleDateDisplay) {
+        @cyaRow(
+            messages(s"sharesDisposalDate.title"),
```

Comment:
you don't need `s` here if you're not interpolating anything

Reply:
Done.